src/openfhe.jl

```diff
@@ -381,3 +385,291 @@ function rotate(sv::SecureVector{<:OpenFHEBackend}, shift; wrap_by)
 
     secure_vector
 end
+
+
+############################################################################################
+# Matrix
+############################################################################################
+
+"""
+    init_matrix_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
+                          shifts, shape)
+
+Generate rotation keys for matrix rotation with `OpenFHE.EvalRotate` using the `private_key` and for the
+rotation indexes in `shifts`. `shifts` is a vector of pair values. The keys are stored in the given
+`context`. Positive shift defines rotation to the right/top, e.g. a rotation with a shift (1, 0):
+[1 2 3; 4 5 6; 7 8 9] -> [7 8 9; 1 2 3; 4 5 6].
+Negative shift defines rotation to the left, e.g. a rotation with a shift (0, 1):
+[1 2 3; 4 5 6; 7 8 9] -> [3 1 2; 6 4 5; 9 7 8].
+"""
+function init_matrix_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
+                               shifts, shape)
```

Comment:
Overall it would be great if you could add some internal comments to this function to explain how and why you add these shifts here. It does not have to be a full write-up (that will come in the paper), but it would be helpful to know at least what the intent of each additionally registered shift is.

Reply:
Thank you very much for the review! I added comments to the `init_matrix_rotation`, hopefully it will be more clear now...

Reply:
Thanks, looks good now!